src/components/Popover/components/PopoverOverlay/PopoverOverlay.tsx

```diff
@@ -114,7 +114,7 @@ export class PopoverOverlay extends PureComponent<PopoverOverlayProps, State> {
         this.clearTransitionTimeout();
         this.exitingTimer = window.setTimeout(() => {
           this.setState({transitionStatus: TransitionStatus.Exited});
-        }, durationFast);
+        }, parseInt(tokens.motion['duration-200'], 10));
```

Comment:
Comparing https://github.com/Shopify/polaris-tokens/blob/main/dist/duration.json and https://github.com/Shopify/polaris-react/blob/v8.0.0-major/src/tokens/_motion.ts:

durationFast is 100ms, so should this be:

```suggestion
        }, parseInt(tokens.motion['duration-100'], 10));
```